File: program/program/find-volume-of-a-cylinder/FindVolumeOfACylinder.cs
using System;

namespace VolumeOfCylinder
{
    // Function to calculate the volume of a cylinder
    static double VolumeOfCylinder(double radius, double height)
    {
        // Calculate the volume using the formula:
        // V = π * r^2 * h
        double volume = Math.PI * Math.Pow(radius, 2) * height;

        // Return the calculated volume
        return volume;
    }

    class Program
    {
        static void Main(string[] args)
        {
            // Example:
            // Calculate the volume of a cylinder with radius 5 and height 10
            double cylinderRadius = 5;
            double cylinderHeight = 10;
            double cylinderVolume = VolumeOfCylinder(cylinderRadius, cylinderHeight);

            Console.WriteLine($"Volume of cylinder = {cylinderVolume}");
        }
    }
}

File: program/program/find-volume-of-a-cylinder/find_volume_of_a_cylinder.dart
import 'dart:math';

double findVolumeOfCylinder(double radius, height) =>
    pi * pow(radius, 2) * height;

void main() {
  print(findVolumeOfCylinder(2, 3)); // will return 37.69911184307752
  print(findVolumeOfCylinder(2.5, 3.746)); // will return 73.55253800217103
}
